app/javascript/react/components/App.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react'
 import {BrowserRouter, Route, Switch} from 'react-router-dom'
+
 import GamesIndexContainer from './GamesIndexContainer'
 import NewGameContainer from './NewGameContainer'
 import GameShowContainer from './GameShowContainer'
-
-
+import UserShowContainer from './UserShowContainer'
 export const App = (props) => {
```

Comment:
Lets add a space between 7 and 8 to separate the import statements from the component